content/includes/header.html

```diff
@@ -20,11 +20,7 @@
 <meta content="https://www.vets.gov" property="og:title">
 {% endif %}
 
-{% if title %}
```

Comment:
I think this needs to stay for SEO purposes. @crwallace can weigh in more. This is a conditional for when there's a `title` property in the front matter. Not sure how much of a difference it makes, but that's why it was added.